pkgs/development/compilers/emscripten/locate_cache.sh

```diff
@@ -0,0 +1,6 @@
+if [ -z "${EM_CACHE}" ]
+then
+  export EM_CACHE="/tmp/$(basename $(realpath $(dirname $(which emcc))/..))_cache"
+  cp -r $(dirname $(which emcc))/../share/emscripten/cache $EM_CACHE
+  chmod +rwX -R $EM_CACHE
```

Comment:
Per @Technius need to diagnose if these two lines can be removed with the patches from #411251 now merged.